.github/workflows/openvex.yml

```diff
@@ -0,0 +1,19 @@
+name: openvex
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  vexctl:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
```

Comment:
I'm afraid running `openvex` for all releases would generate unnecessary noise, since we generally only maintain the latest release branches. Can we restrict this to that only? cc @mrueg

Reply:
This is restricted only to a pushed tag. See line 7

Reply:
I think it might be better to run on release creation. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Reply:
+1, better to run on releases than tags.